requirements-cwl.txt

```diff
@@ -1,2 +1,4 @@
 cwltool==3.1.20211107152837
 galaxy-tool-util
+ruamel.yaml>=0.17.16,<0.17.17
```

Comment:
Is this the same as `ruamel.yaml==0.17.16`?

Reply:
I need the lower bound because older versions are not pickle-compatible with newer ones, and the upper bound because my `pip` wasn't able to find a correct package solution without it, since `cwltool` has an upper bound. I anticipate being able to bump the upper bound when cwltool updates.

On the other hand, pinning an exact version will stop us from ever having the pickle incompatibility problem again.